lib/pure/strutils.nim

```diff
@@ -1442,30 +1512,151 @@ proc formatFloat*(f: float, format: FloatFormatMode = ffDefault,
   ## of significant digits to be printed.
   ## `precision`'s default value is the maximum number of meaningful digits
   ## after the decimal point for Nim's ``float`` type.
-  result = formatBiggestFloat(f, format, precision, decimalSep)
+  ## `trim` can be used to remove trailing zeros.
+  result = formatBiggestFloat(f, format, precision, decimalSep, trim)
+
+proc formatEng*(f: BiggestFloat,
+                precision: range[0..32] = 10,
+                decimalSep = '.',
+                trim: bool = true,
+                siPrefix: bool = false,
+                unit: string = nil): string =
+  ## See documentation under forward declaration (above)
```

Comment:
Remove this doc comment and the export marker please. The doc gen doesn't like this iirc.